staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured/unstructured.go

```diff
@@ -239,6 +239,42 @@ func (u *Unstructured) SetGeneration(generation int64) {
 	u.setNestedField(generation, "metadata", "generation")
 }
 
+func (u *Unstructured) SetObservedGeneration(observedgeneration int64) {
```

Comment:
`Unstructured` does not really have a concept of `status` and `spec` and these are not known and "standard" fields in it. In fact, we do not use the concept of `ObservedGeneration` anywhere. It is possible to have this field but it holds no special significance to us.

So I think we shouldn't add it here, since this is more for known fields (like metadata).

You can add these in `etcd_test.go` though - the tests would be easier to read that way! :+1:

Reply:
Done